apps/api/src/domain/models/audio-visual/shared/commands/transcripts/translate-line-item/line-item-translated.event.ts

```diff
@@ -0,0 +1,41 @@
+import { AggregateType, LanguageCode } from '@coscrad/api-interfaces';
+import buildDummyUuid from '../../../../../../../domain/models/__tests__/utilities/buildDummyUuid';
+import { dummyDateNow } from '../../../../../../../domain/models/__tests__/utilities/dummyDateNow';
+import { CoscradDataExample } from '../../../../../../../test-data/utilities';
+import { DTO } from '../../../../../../../types/DTO';
+import { CoscradEvent } from '../../../../../../common';
+import { BaseEvent } from '../../../../../shared/events/base-event.entity';
+import { TranslateLineItem } from './translate-line-item.command';
+
+export type LineItemTranslatedPayload = TranslateLineItem;
+
+const testEventId = buildDummyUuid(545);
+
+@CoscradDataExample<LineItemTranslated>({
+    example: {
+        // TODO can we avoid this as it is automatically set by the constructor? It's only here for consistency with other DTOs.
```

Comment:
Really good point. I should make a tech debt story for this. I don't want to rewrite lots of code on a regular feature branch.

Reply:
[Here](https://coscrad.atlassian.net/browse/CWEBJIRA-66) is that story.